corehq/apps/reports_core/filters.py

```diff
@@ -229,11 +229,12 @@ def __init__(self, name, field, required, datatype, label, show_all, url_generat
         self.url_generator = url_generator
 
     def value(self, **kwargs):
-        choice = kwargs[self.name]
-        if choice:
-            typed_choice = transform_from_datatype(self.datatype)(choice)
-            return Choice(typed_choice, choice)
-        return Choice(SHOW_ALL_CHOICE, '')
+        selection = kwargs[self.name]
```

Comment:
not a blocker, but I think I'd prefer `kwargs.get(self.name)` here in case it's not sent in